src/main/java/com/chaos131/poses/MirroredDrivePose.java

```diff
@@ -116,6 +118,20 @@ public double getDistanceFromLocation(Pose2d robotpose) {
      * (Override this when using unit testing since the call to DriverStation will throw an exception)
      */
     protected Alliance getCurrentAlliance() {
+        // A mostly OSX specific bugfix that lets the field pose flip based on what
+        // the simulation interface says the robot is a team member of
+        if (HALUtil.getHALRuntimeType() == HALUtil.RUNTIME_SIMULATION) {
```

Comment:
What was the bug? DriverStation.getAlliance() doesn't work on OSX?

Reply:
The Javadocs for getAlliance() don't mention that it supports the simulation as a datasource. Only that it uses the FMS, HAL and DriverStation as data sources. None of which I can use on my osx laptop at home. So I thought I had to swap over to the simulation one to pull the right data. Apparently it's fine though, the bug I was tracking had to do with PathPlanner's API and me not reading all of it yet.